src/object/cli_obj.c

```diff
@@ -74,6 +74,7 @@ struct obj_auxi_args {
 	tse_task_t			*obj_task;
 	int				 opc;
 	int				 result;
+	daos_epoch_t			 epoch;
```

Comment:
we probably need consider use union in this structure, the size is too big now, and almost hit the maxim of task buffer size.